app/pages/nbtc-mint/ExpandableRedeemDetails.tsx

```diff
@@ -0,0 +1,108 @@
+import { Info, CheckCircle } from "lucide-react";
+import { type RedeemRequestResp, RedeemRequestStatus } from "@gonative-cc/sui-indexer/models";
+import { AnimatedHourglass } from "~/components/ui/AnimatedHourglass";
+import { Tooltip } from "~/components/ui/tooltip";
+import { formatNBTC } from "~/lib/denoms";
+
+interface ExpandableRedeemDetailsProps {
+	transaction: RedeemRequestResp;
+}
+
+export function ExpandableRedeemDetails({ transaction }: ExpandableRedeemDetailsProps) {
+	const operationDate = new Date(transaction.created_at);
+
+	const isPending = transaction.status === RedeemRequestStatus.Pending;
+	const isProposed = transaction.status === RedeemRequestStatus.Proposed;
+	const isSigned = transaction.status === RedeemRequestStatus.Signed;
+	const isBroadcasted = transaction.status === RedeemRequestStatus.Broadcasted;
+	const isSolved = transaction.status === RedeemRequestStatus.Solved;
+
+	const isUTXOSelectionProposed = isProposed || isSigned || isBroadcasted || isSolved;
+	const isTxnSigned = isSigned || isBroadcasted || isSolved;
+	const isBitCoinTxBroadcasted = isBroadcasted || isSolved;
+
+	return (
+		<div className="bg-base-100 border-base-300 p-6">
+			<div className="space-y-4">
+				<div className="text-base-content/70 text-sm">
+					<span className="font-semibold">Redeem request created:</span>{" "}
+					{operationDate.toLocaleString()}
+				</div>
+
+				<div className="divider my-2"></div>
+
+				<div className="flex items-center gap-2 text-sm">
+					{isPending ? (
+						<AnimatedHourglass size="md" />
+					) : (
+						<CheckCircle size={16} className="text-success shrink-0" />
+					)}
+					<span>{isPending ? "Redeem request pending" : "Redeem request created"}</span>
+				</div>
+
+				<div className="flex items-center gap-2 text-sm">
+					{isUTXOSelectionProposed ? (
+						<CheckCircle size={16} className="text-success shrink-0" />
+					) : isPending ? (
+						<AnimatedHourglass size="md" />
+					) : null}
+					<span>
+						{isUTXOSelectionProposed ? "UTXO selection proposed" : "Awaiting UTXO selection"}
```

Comment:
Remove good animation, and show the status with a proper stage. You can show it was a simple list, or similar to how we display minting stages.
- pending
- solving
- solved
- signing
- broadcasted

Stages completed should be marked with a checkmark. 
The current stage - with a timer. 
Future stages - can be great

Reply:
I have updated the stages in the UI.